sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/RangeActionSensitivityProvider.java

```diff
@@ -68,7 +68,7 @@ private boolean willBeKeptInSensi(Generator gen) {
         return gen.getTerminal().isConnected() && gen.getTerminal().getBusBreakerView().getBus().isInMainSynchronousComponent();
     }
 
-    private SensitivityVariable defaultSensitivityVariable(Network network) {
+    protected SensitivityVariable defaultSensitivityVariable(Network network) {
```

Comment:
could be then moved to the LoadFlowProvider is this class now extends LoadFlowProvider

Reply:
yeah I had to move a few methods